src/js/constants/deviceConstants.js

```diff
@@ -37,9 +38,9 @@ module.exports = {
   SET_DEVICE_LIMIT: 'SET_DEVICE_LIMIT',
 
   EXTERNAL_PROVIDER: {
-    amazon: { article: 'an', icon: AWS, title: 'AWS IoT core', provider: 'amazon', managementUrl: '/v1/aws-iot-manager/settings', enabled: false },
-    azure: { article: 'an', icon: Azure, title: 'Azure IoT Hub', provider: 'azure', managementUrl: '/v1/azure-iot-manager/settings', enabled: true },
-    google: { article: 'a', icon: GCP, title: 'Cloud IoT Core', provider: 'google', managementUrl: '/v1/gcp-iot-manager/settings', enabled: false }
+    amazon: { article: 'an', icon: AWS, title: 'AWS IoT core', provider: 'amazon', managementUrl: iotManagerBaseURL + '/settings', enabled: false },
+    azure: { article: 'an', icon: Azure, title: 'Azure IoT Hub', provider: 'azure', managementUrl: iotManagerBaseURL + '/settings', enabled: true },
```

Comment:
I suppose we need to change the Settings schema too, since 'connection_string' won't be a universal authentication method.

Reply:
For sure, but right now I don't know how to change it without breaking it. I think the decision is up to you, how to manage the API namespace in iot-manager. We'll need a follow-up PR once it's defined.